### PR TITLE
Pa11y dependencies

### PR DESCRIPTION
**Pull request by kkoskelin**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -71,7 +71,7 @@ jobs:
             - '.git'
       - run:
           name: NPM - Install
-          command: npm set progress=false && npm ci && npm rebuild node-sass
+          command: npm set progress=false && npm ci && npm rebuild node-sass && npm ci --prefix=web-client/pa11y/
```
Comment (sutt0n) [inline .circleci/config.yml]: nit: change `command: npm ... && npm ... && npm ...` to:

```          
          command: |
            npm set progress=false
            npm ci
            npm rebuild node-sass
            npm ci --prefix=web-client/pa11y/
```

**Review by ericsorenson — APPROVED** [review]

File: web-api/start-dynamo.sh [hunk 2]
```diff
@@ -11,4 +11,4 @@ if [ ! -e ".dynamodb/dynamo.tar.gz" ]; then
 fi
 
 # start dynamo
-cd .dynamodb && java -Djava.library.path=.dynamodb/DynamoDBLocal_lib -jar DynamoDBLocal.jar -sharedDb
+cd .dynamodb && java -Djava.library.path=.dynamodb/DynamoDBLocal_lib -Xms512m -Xmx1g -jar DynamoDBLocal.jar -sharedDb
```
Comment (codyseibert) [inline web-api/start-dynamo.sh]: did we want this hard coded here still since we set it manually in config.yml?

File: .circleci/config.yml [hunk 3]
```diff
@@ -130,9 +142,13 @@ jobs:
...
       - image: amazon/dynamodb-local
         command: ['-jar', 'DynamoDBLocal.jar', '-sharedDb']
-      - image: elastic/elasticsearch:7.5.2
         environment:
           discovery.type: single-node
+          JAVA_OPTS: '-Xms512m -Xmx1g'
+      - image: elastic/elasticsearch:7.8.1
+        environment:
+          discovery.type: single-node
+          ES_JAVA_OPTS: '-Xms512m -Xmx512m'
```
Comment (ericsorenson) [inline .circleci/config.yml:151]: 🎉

File: web-api/start-dynamo.sh (hunk 2 above)
Comment (kkoskelin) [reply web-api/start-dynamo.sh]: Good call, this can come out, I think

**Review by ericsorenson — APPROVED** [review]

**Pull request merged**